components/date-picker/RangePicker.tsx

```diff
@@ -267,6 +267,7 @@ export default class RangePicker extends React.Component<any, any> {
             disabled={props.disabled}
             readOnly
             value={(start && start.format(props.format)) || ''}
+            id={props.id && `${props.id}-start`}
```

Comment:
We should add id in the [wrapper](https://github.com/ant-design/ant-design/pull/8598/files#diff-0f7c946177eb7f19b2586e4cb67ee699R265),  instead inputs..

Reply:
That would mean we would have to search for the first and second input inside the wrapper:
findById(myid)->find(input)[**nth**]

What do you think of adding an additional id to the wrapper, without the start and end postfix?
findById(myid)->find(input#myid-start)
findById(myid)->find(input#myid-end)

Reply:
@yesmeck What your opinion. I will release a new version soon.

Reply:
Agree with @benjycui